### DISCUSSION TITLE
Some small feedback about recent label changes

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#6784] [Projects and Issues]
Just now noticed the colour change for closed and resolved issues and after [reading through the blog post](https://github.blog/changelog/2021-10-26-updates-to-our-issue-status-icons-and-colors/) and the linked roadmap entry am I here to quickly give my general feedback on this.

- The usage of purple to mark resolved issues is not really a good colour choice. The usage of the green (or in the case of colour blind, blue) colour would've been a better solution here.
- Open issue should have a red (or in the case of colour blind, orange) colour since they are in fact issues.
  - The purple colour could be used here to indicate issues that are more enhancements, questions, feedback, etc.

In general am I find with a colour change of the issue states. In fact did I plan to suggest this a while back but it slipped my mind and I forgot it over time.
It's just the colour that was selected that I can't really get behind, especially since the colour for open issues is still a positive green, which really doesn't fit the purpose of what an open issue is: A notice of an issue...

Hope to see this being improved further in the future. 👋

**Comment by ouuan** [comment]
I suggest changing the color of "Closed with won't fix or duplicate" from grey to red, and keeping the open icon green, closed icon purple. The key consideration is to keep it consistent with the colors of the PR icons.

**Comment by ouuan** [comment]
> especially since the colour for open issues is still a positive green, which really doesn't fit the purpose of what an open issue is: A notice of an issue...

I personally don't find it confusing... I think it's more important to keep the colors consistent with the pull requests. However, you may think like this: A bug is a negative thing, while an issue is a positive thing that can help you fix the bug.

**Reply by ouuan** [reply]
And a common practice is to make the "bug" label red.
